package.json

```diff
@@ -352,7 +354,7 @@
     "react-beautiful-dnd": "^13.0.0",
     "react-refresh": "^0.9.0",
     "react-router-dom": "^5.2.0",
-    "react-select": "^3.1.0",
+    "react-select": "^4.3.0",
```

Comment:
Should we also update `@types/react-select` to latest?

Reply:
Updated types to latest version.